contracts/phoenix_fee_contract/v1/ergoscript/phoenix_v1_hodlcoin_fee.es

```diff
@@ -25,60 +25,58 @@
     // ===== Context Variables (@) ===== //
     // None
 
-    val feeDenom = 100L
-    val devPercentageNum: Long = 60L
-    val phoenixPercentageNum: Long = 40L
-
     // ===== Relevant Variables ===== //
-    val dev1Address: SigmaProp                  = PK("9hHondX3uZMY2wQsXuCGjbgZUqunQyZCNNuwGu6rL7AJC8dhRGa")
-    val dev2Address: SigmaProp                  = PK("9gnBtmSRBMaNTkLQUABoAqmU2wzn27hgqVvezAC9SU1VqFKZCp8")
-    val dev3Address: SigmaProp                  = PK("9iE2MadGSrn1ivHmRZJWRxzHffuAk6bPmEv6uJmPHuadBY8td5u")
-    val phoenixAddress: SigmaProp               = PK("9iPs1ujGj2eKXVg82aGyAtUtQZQWxFaki48KFixoaNmUAoTY6wV")
-    val minerFeeErgoTreeBytesHash: Coll[Byte]   = fromBase16("e540cceffd3b8dd0f401193576cc413467039695969427df94454193dddfb375")
+    val minerFeeErgoTreeBytesHash: Coll[Byte] = fromBase16("e540cceffd3b8dd0f401193576cc413467039695969427df94454193dddfb375")
+    
+    val feeDenom: Long   = 100L
+    val brunoNum: Long   = 25L
+    val pulsarzNum: Long = 25L
+    val phoenixNum: Long = 25L
+    val kushtiNum: Long  = 15L
+    val krasNum: Long    = 10L
+    
+    val brunoAddress: SigmaProp   = PK("9gnBtmSRBMaNTkLQUABoAqmU2wzn27hgqVvezAC9SU1VqFKZCp8")
+    val pulsarzAddress: SigmaProp = PK("9hHondX3uZMY2wQsXuCGjbgZUqunQyZCNNuwGu6rL7AJC8dhRGa")
+    val phoenixAddress: SigmaProp = PK("9iPs1ujGj2eKXVg82aGyAtUtQZQWxFaki48KFixoaNmUAoTY6wV")
+    val kushtiAddress: SigmaProp  = PK("9iE2MadGSrn1ivHmRZJWRxzHffuAk6bPmEv6uJmPHuadBY8td5u")
+    val krasAddress: SigmaProp    = PK("")
```

Comment:
Address missing here.

Reply:
Yeah we are waiting for him to get back to us with an address he wants to use. Will update with new commit once we get it.